packages/cli/generators/example/index.js

```diff
@@ -59,6 +59,7 @@ const EXAMPLES = {
   'todo-jwt': g.f('A modified Todo application with JWT authentication.'),
   webpack: g.f('An example to bundle @loopback/core using webpack.'),
   graphql: g.f('An example to demonstrate GraphQL integration.'),
+  socketio: g.f('A basic implementation of Socket.IO'),
```

Comment:
maybe `socket.io` instead of `Socket.IO`? 
```suggestion
  socketio: g.f('A basic implementation of socker.io'),
```

Reply:
It looks like the official name of the library is Socket.IO. You can check [Wikipedia](https://en.wikipedia.org/wiki/Socket.IO) or [npm package](https://www.npmjs.com/package/socket.io).
I think it is the same as GraphQL, Express, Prometheus, and LoopBack.

Reply:
@dhmlau Is there anything else I need to do to close this pull request?